### PR TITLE
 lint README and add ToC for repo

### PR DESCRIPTION
**Pull request by zanetworker**
The following changes has been made to the README:
 - add ToC
 - format bash scripts in markdown
 - add description of what this repo is for

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,35 @@
 # Gardener Setup Scripts
 
-This is the installation manual for a simple Gardener setup. It is part of the [landscape-setup-template](https://github.com/gardener/landscape-setup-template) project. You can find further information there.
-
-
+This repo serves as the installation manual for a simple Gardener setup. Basically, it explains how you can create a simple cluster with project [Kubify](https://github.com/gardener/kubify) and walks you through the steps required to bootstrap a Gardener cluster in an automated manner (if you just want to run a script) or in a step-by-step manner for those who seek more understanding of whats going on under the hood.
```
Comment (msohn) [inline README.md]: No, this README is the manual, the repo contains the installer, and the landscape-setup-template serves as a configuration template for this installer.

typo: s/whats/what's/

File: README.md [hunk 2]
```diff
@@ -1,8 +1,35 @@
 # Gardener Setup Scripts
 
-This is the installation manual for a simple Gardener setup. It is part of the [landscape-setup-template](https://github.com/gardener/landscape-setup-template) project. You can find further information there.
-
-
+This repo serves as the installation manual for a simple Gardener setup. Basically, it explains how you can create a simple cluster with project [Kubify](https://github.com/gardener/kubify) and walks you through the steps required to bootstrap a Gardener cluster in an automated manner (if you just want to run a script) or in a step-by-step manner for those who seek more understanding of whats going on under the hood. 
+
+This repo is also part of the [landscape-setup-template](https://github.com/gardener/landscape-setup-template) project. You can find further information there.
```
Comment (msohn) [inline README.md]: not really,  the landscape-setup-template serves as a configuration template for this installer

File: README.md [hunk 3]
```diff
@@ -1,8 +1,35 @@
...
+
+- [Gardener Setup Scripts](#gardener-setup-scripts)
+- [Prerequisites](#prerequisites)
+- [Gardener Installation](#gardener-installation)
+  - [TLDR](#tldr)
+  - [Step 1: Clone the Repositories and get Dependencies](#step-1-clone-the-repositories-and-get-dependencies)
+  - [Step 2: Configure the Landscape](#step-2-configure-the-landscape)
+  - [Step 3: Build and Run Docker Container](#step-3-build-and-run-docker-container)
+  - [Step 4: Create a Kubernetes Cluster via Kubify](#step-4-create-a-kubernetes-cluster-via-kubify)
+  - [<a name="workaround"></a>Step 4.5: Workaround (Automated)](#a-name%22workaround%22astep-45-workaround-automated)
```
Comment (msohn) [inline README.md]: no link is rendered for this entry

File: README.md [hunk 4]
```diff
@@ -1,8 +1,35 @@
...
+  - [Step 5-9: Gardener Setup (Automated)](#step-5-9-gardener-setup-automated)
+  - [<a name="access_dashboard"></a>Accessing the Dashboard](#a-name%22accessdashboard%22aaccessing-the-dashboard)
```
Comment (msohn) [inline README.md]: no link is rendered for this entry

File: README.md [hunk 5]
```diff
@@ -1,8 +1,35 @@
 # Gardener Setup Scripts
 
-This is the installation manual for a simple Gardener setup. It is part of the [landscape-setup-template](https://github.com/gardener/landscape-setup-template) project. You can find further information there.
-
-
+This repo serves as the installation manual for a simple Gardener setup. Basically, it explains how you can create a simple cluster with project [Kubify](https://github.com/gardener/kubify) and walks you through the steps required to bootstrap a Gardener cluster in an automated manner (if you just want to run a script) or in a step-by-step manner for those who seek more understanding of whats going on under the hood. 
+
+This repo is also part of the [landscape-setup-template](https://github.com/gardener/landscape-setup-template) project. You can find further information there.
+
+Finally, the table of contents can be found below in case you want to jump quickly to your favorite part:
```
Comment (msohn) [inline README.md]: why do we need a long sentence to explain the ToC ?

File: README.md [hunk 6]
```diff
@@ -43,12 +69,14 @@ k8s/bin/tf destroy -force
 setup/cleanup.sh
 ```
 
+If not, please follow our step-by-step guide below.
```
Comment (msohn) [inline README.md]: no need for "please" here, how about this :

Otherwise follow our step-by-step guide below.

**Review by msohn — DISMISSED** [review]

**Comment by zanetworker** [comment]
@msohn thank you for reviewing. I have updated the pull-request based on your comments.

**Comment by Diaphteiros** [comment]
Has been moved to https://github.com/gardener/landscape-setup/pull/30

**Pull request closed without merging**